python/tank_vendor/shotgun_deploy/manager.py

```diff
@@ -205,25 +205,15 @@ def bootstrap_engine(self, engine_name, entity=None):
         return engine
 
 
-    def _bootstrap_sgtk(self, engine_name, entity):
-        """
-        Create an sgtk instance for the given engine and entity.
-
-        If entity is None, the method will bootstrap into the site
-        config. This method will attempt to resolve the config according
-        to business logic set in the associated resolver class and based
-        on this launch a configuration. This may involve downloading new
-        apps from the toolkit app store and installing files on disk.
+    def get_config(self, engine_name, entity):
```

Comment:
I made it public since I needed a way to get the path to core at a very high level. Maybe there's a different/better way to do it without making this public? Maybe just a method to return the config path given the same args?